_includes/nav.html

```diff
@@ -45,7 +45,9 @@
 </li>
 <li><a href="/news/">News</a></li>
 <li class="has-button">
-  <a class="button hollow theme-festival" href="{{ site.ticket_link }}">
-    Tickets On Sale!
+  <a class="button hollow theme-festival" href="/conduct/">
```

Comment:
This is a great idea, but we aren't there yet with tutorials still selling. Can we remove / queue this up for a few days before?

Reply:
@jefftriplett Should we change it to "Tutorials still on sale" then?

Reply:
Maybe "Limited Tickets Left" or something better worded instead?